storage/innobase/xtrabackup/src/xb_dict.cc

```diff
@@ -228,7 +228,26 @@ b - std::vector<dict_table_t*>, empty on errors */
 static xb_dict_tuple dict_load_tables_from_space_id_wrapper(
     space_id_t space_id, table_id_t table_id) {
   fil_space_t *space = fil_space_get(space_id);
+
   if (space == nullptr) {
+    dberr_t err = fil_xb_get_tablespace_error(space_id);
+    /* Check if tablespace errored out when it was loaded. If yes, we abort
+    the prepare. If no, we consider the tablespace as dropped */
+    if (err != DB_ERROR_UNSET) {
+      std::string tablespace_name;
+      bool found = fil_system_get_file_by_space_id(space_id, tablespace_name);
+      xb::error() << "Tablespace " << (found ? tablespace_name : "")
+                  << " with space_id " << space_id
+                  << " is required for transaction rollback but it was not"
+                  << " loaded because of DB_ error " << err;
```

Comment:
Do we print DB_ERROR or INT here ?

Reply:
unfortunately it is a number. We dont have a string mapping from the c style enum.